src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/HelpWithFeesDocumentService.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.finrem.caseorchestration.client.DocumentClient;
+import uk.gov.hmcts.reform.finrem.caseorchestration.config.DocumentConfiguration;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseDocument;
+
+@Service
+@Slf4j
+public class HelpWithFeesDocumentService extends AbstractDocumentService {
```

Comment:
this will become obsolete once recent refactor is megred in - `AbstractDocumentService` was removed. Perhaps merge the branch `refactor-document-service` in?